.github/actions/build-publish-docker/action.yml

```diff
@@ -117,6 +117,12 @@ runs:
           build-args: ${{ inputs.build-args }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache-new
+    - name: Post image name to PR comments
+      if: steps.build-and-push-pr.outcome == 'success'
```

Comment:
This is even better! 🙂  But then you need to set `id=build-and-push-pr` on the previous step I think?

Reply:
thank you very much for the valuable notes! all done